src/index.html

```diff
@@ -11,6 +11,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search search_big" data-qa="big">
+      <div class="search_big__line">
```

Comment:
let's simplify the code and remove this wrapper along with its styles

Reply:
done